src/kOS/Suffixed/OrbitInfo.cs

```diff
@@ -1,4 +1,4 @@
-﻿using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation;
```

Comment:
Go home, git diff, you're drunk.  There's literally no diff between these two lines.  Not even in whitespace.